src/inmanta/ast/statements/call.py

```diff
@@ -161,12 +129,99 @@ def pretty_print(self):
                 chain(
                     (a.pretty_print() for a in self.arguments),
                     ("%s=%s" % (k, v.pretty_print()) for k, v in self.kwargs.items()),
-                    ("**%s" % kwargs.pretty_print() for kwargs in self.wrapped_kwargs),
+                    ("%s" % kwargs.pretty_print() for kwargs in self.wrapped_kwargs),
                 )
             ),
         )
 
 
+class Function:
+    def __init__(self, ast_node: FunctionCall) -> None:
+        self.ast_node: FunctionCall = ast_node
+
+    def call(self, args, kwargs) -> object:
```

Comment:
I would prefer this method to be called call_direct (to be in line with execute_direct)